Main.java

```diff
@@ -0,0 +1,722 @@
+package Main.java;
+
+import javax.annotation.processing.SupportedSourceVersion;
+import java.time.LocalDate;
+import java.util.Arrays;
+import java.util.Scanner;
+
+class Client {
+    private String nume;
+    private String prenume;
+    private String cnp;
+    private LocalDate dataImprumutarii_Returului;
+    private Carte carteImprumutata_Returnata;
+
+    // Getters and Setters
+
+    public String getNume() {
+        return nume;
+    }
+    public void setNume(String nume) {
+        this.nume = nume;
+    }
+
+    public String getPrenume() {
+        return prenume;
+    }
+    public void setPrenume(String prenume) {
+        this.prenume = prenume;
+    }
+
+    public String getCnp() {
+        return cnp;
+    }
+    public void setCnp(String cnp) {
+        this.cnp = cnp;
+    }
+
+    public LocalDate getDataImprumutarii_Returului() {
+        return dataImprumutarii_Returului;
+    }
+    public void setDataImprumutarii_Returului(LocalDate dataImprumutarii_Returului) {
+        this.dataImprumutarii_Returului = dataImprumutarii_Returului;
+    }
+
+    public Carte getCarteImprumutata_Returnata() {return carteImprumutata_Returnata;}
+    public void setCarteImprumutata_Returnata(Carte carteImprumutata_Returnata) {this.carteImprumutata_Returnata = carteImprumutata_Returnata;}
+
+    Client(String nume, String prenume, String cnp, Carte carteImprumutata_Returnata){
+        this.nume = nume;
+        this.prenume = prenume;
+        this.cnp = cnp;
+        this.dataImprumutarii_Returului = LocalDate.now();
+        this.carteImprumutata_Returnata = carteImprumutata_Returnata;
+    }

```

Comment:
constructors should come after fields, as a best practice, before setters and getters